ocaml/xapi/xapi_host.ml

```diff
@@ -1474,7 +1474,7 @@ let replace_host_certificate ~__context ~type' ~host
   List.iter (Db_util.remove_cert_by_ref ~__context) old_certs ;
   let task = Context.get_task_id __context in
   Db.Task.set_progress ~__context ~self:task ~value:1.0 ;
-  Xapi_mgmt_iface.reconfigure_stunnel ~__context
```

Comment:
This allows to delete the function reconfigure_stunnel, the other user of the function is never called (restart_stunnel in xapi/cert_distrib.ml)

Reply:
The existing code for Stunnel reconfiguration is quite complex. Is all we ever need this simple reload?

Reply:
If it's for the stunnel server only: yes